src/jest.mli

```diff
@@ -14,8 +14,12 @@ module Runner (A : Asserter) : sig
   external describe : string -> (unit -> unit) -> unit = "" [@@bs.val]
 
   external beforeAll : (unit -> unit) -> unit = "" [@@bs.val]
+  val beforeAllAsync : ((_ A.t -> unit) -> unit) -> unit
```

Comment:
Why does finish take an assertion? Is there some use case for that?

Also, it would be nice if there was an optional `timeout` argument.

Reply:
Fixed (assertion) and added optional `timeout` argument.